### DISCUSSION TITLE
Marketing Team meeting planning - 2022-03-09

### DISCUSSION THREAD

**Discussion by nalininsbs** [WordPress/Marketing-Team#10] [General]
Test in the discussion area in the new repo to see if it shows for everyone, and if they can add to it

**Comment by murillol** [comment]
[Agenda: Global Marketing Team, 2 March 2022](https://make.wordpress.org/marketing/?p=8792&preview=true)

Join the team at 15:00 UTC in the [Marketing channel on the Make WordPress Slack](https://wordpress.slack.com/archives/C0GKJ7TFA). Async contribution is welcome too.

1.  **Welcome**
2. **Getting to know each other**
    a. Attendees
    b. Social question of the week
3. [**Notes from the Last Meeting**](https://make.wordpress.org/marketing/2022/03/01/notes-global-marketing-team-23-february-2022/)
  a. Volunteer call for notetakers.
4. **Project Updates**
    a. People of WordPress
    b. GitHub updates and links
5. **Diary/Calendar Dates**
        Meeting time  
        Follow this Google [Calendar with meetings, collaborations, and social activities](https://calendar.google.com/calendar/b/2?cid=NDQyN2g1cnZrN2EwYXA5Y3VpNDI5OGYzY3NAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ). We also list important dates in this [Forthcoming Diary Dates](https://make.wordpress.org/marketing/2021/01/13/forthcoming-diary-dates/) post.
6. **Open Floor**

If you have any additions in mind, please comment below.

--------

Above is the proposed Agenda (not on the P2 yet as a draft), other items which might be included:

- [WordPress 5.9.1 ](https://wordpress.org/news/2022/02/wordpress-5-9-1-maintenance-release/)
- [Proposal to Start a News blog on developer.WordPress.org](https://make.wordpress.org/core/2022/02/25/proposal-to-start-a-news-blog-on-developer-wordpress-org/)
- [FSE Program Testing Call #12: Hyping Headers](https://make.wordpress.org/test/2022/02/23/fse-program-testing-call-12-hyping-headers/)

@nalininsbs @eidolonnight

**Reply by murillol** [reply]
It is now on the P2 as a draft, appreciate your comments

**Comment by murillol** [comment]
**Update**: [script draft is here](https://docs.google.com/document/d/1Xkr09KmocCbMIzFNYp8KRQnsqZHUB7_HBjq8XD8JcxE/edit?usp=sharing)
